http/src/main/java/com/redhat/lightblue/client/http/transport/SelfClosingApacheHttpClient.java

```diff
@@ -0,0 +1,46 @@
+package com.redhat.lightblue.client.http.transport;
+
+import com.redhat.lightblue.client.LightblueClientConfiguration;
+import com.redhat.lightblue.client.request.LightblueRequest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+
+/**
+ * A simple to use though inefficient http client which does not require closing. Useful for simple
+ * applications or one off requests.
+ */
+public class SelfClosingApacheHttpClient implements HttpClient {
```

Comment:
This should be deprecated or removed I think. Not sure if there's any reason to use this over the java.net implementation.